tests/scripts/remote.sh

```diff
@@ -4,4 +4,5 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source ${SCRIPT_DIR}/.definitions.sh
 source ${SCRIPT_DIR}/.local.sh
 
-ssh -i ${private_key} ${instance_hostname} "${@}"
+# keep alive 60sec and timeout after 30 tries
+ssh -o ServerAliveInterval=60 -o ServerAliveCountMax=30 -i ${private_key} ${instance_hostname} "${@}"
```

Comment:
Question -- why is this required?

Reply:
default ssh server timeout 60sec
_kubectl wait or sleep  :
 ssh might drop the connection if there's no activity between the client and server ( default timeout 60sec)_